### PR TITLE
various translations for raidboss & ffxiv-eureka web overlay

### PR DESCRIPTION
**Pull request by DrLippe**

### CODE REVIEW

File: ui/eureka/eureka.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  <script>
+    document.addEventListener("onZoneChangedEvent", function (e) {
+	    if (e.detail.zoneName == 'Eureka Anemos') 
+	      document.getElementById("eurekaId").style.display = "block";
+	    else
+	  	document.getElementById("eurekaId").style.display = "none";
+    });
+    document.addEventListener("onLogEvent", function(e) {
+      for (var i = 0; i < e.detail.logs.length; i++) {
+        var eurekaId = e.detail.logs[i].match('ffxiv-eureka.com/(.{6})');
```
Comment (quisquous) [inline ui/eureka/eureka.html]: Oh neat, I was thinking about doing something like this myself, but I found the eureka tracker to be way too big for my screen.  I was going to try to pull out that information into a different format somehow.  Do you run ffxiv at a really high screen resolution?

At the very least, can you pull this eureka stuff out into a separate pull request? I generally like to have pull requests (and commits) be logically related.  If you're doing a bunch of translations, those can all go together.  But translations and eureka are separate in my mind.

File: ui/raidboss/data/triggers/o7s.js [hunk 2]
```diff
@@ -48,107 +102,199 @@
...
     {
       id: 'O7S Abandonment',
-      regex: / 1A:(\y{Name}) gains the effect of Abandonment/,
+      regex: / 1A:(\y{Name}) gains the effect of (?:孤独感|Abandonment|Verlassen|Isolement)/,
       condition: function(data, matches) { return data.me == matches[1]; },
-      alertText: 'Abandonment: stay middle',
-      tts: 'abandonment',
+      alarmText: function(data) {
+		  return {
+			  en: 'Abandonment: go mid',
```
Comment (quisquous) [inline ui/raidboss/data/triggers/o7s.js]: Please use spaces instead of tabs, here and elsewhere.

File: ui/raidboss/data/triggers/o6s.js [hunk 3]
```diff
@@ -1,35 +1,179 @@
...
+		      };
         if (data.role == 'healer')
-          return 'Buster on ' + data.ShortName(matches[1]);
+          return {
+			      en: 'Buster on ' + data.ShortName(matches[1]),
+			      de: 'Buster auf ' + data.ShortName(matches[1]),
+		      };
       },
       tts: function(data, matches) {
         if (matches[1] == data.me)
```
Comment (quisquous) [inline ui/raidboss/data/triggers/o6s.js]: Please use {} for conditionals that run over one line.

File: ui/raidboss/data/triggers/o5s.js [hunk 4]
```diff
@@ -36,7 +36,7 @@
     {
       locale: 'fr',
       replaceSync: {
-        'Engage!': "À l'attaque!",
+        'Engage!': "À l'attaque !",
```
Comment (quisquous) [inline ui/raidboss/data/triggers/o5s.js]: No space here before the exclamation point for French engage, here and elsewhere.  See comments in #57 for more details.

File: ui/raidboss/data/triggers/o5s.js [hunk 5]
```diff
@@ -58,49 +58,37 @@
...
         data.StopCombat();
       },
     },
     {
       id: 'O5S Doom Strike',
-      regex: /14:28B1:Phantom Train starts using Doom Strike on (\y{Name})/,
-      regexDe: /14:28B1:Phantomzug starts using Vernichtungsschlag on (\y{Name})/,
-      regexFr: /14:28B1:Train Fantôme starts using Frappe Létale on (\y{Name})/,
-      regexJp: /14:28B1:魔列車 starts using 魔霊撃 on (\y{Name})/,
+      regex: /14:28B1:(?:魔列車|Phantom Train|Phantomzug|Train Fantôme) starts using (?:魔霊撃|Doom Strike|Vernichtungsschlag|Frappe Létale) on (\y{Name})/,
```
Comment (quisquous) [inline ui/raidboss/data/triggers/o5s.js]: This undos what I did in 4022e58.  I intentionally want to have different regexes for different languages here.  I think it makes regexes more readable, makes cactbot less complicated to not have to handle two language settings elsewhere, and also makes it possible to write automated tools to know what translations are missing.  I just don't think folks using different languages for the ACT plugin vs the ffxiv games is something I want to support.

(It looks like you had your own local version of the file that and didn't make changes based on the latest version on github, so you have a lot more changes in this file than I think there should be.)

File: ui/raidboss/data/triggers/o6s.js [hunk 6]
```diff
@@ -1,35 +1,179 @@
...
-      regex: / 14:2829:Demon Chadarnook starts using Demonic Shear on (\y{Name})/,
+      regex: / 14:2829:(?:チャダルヌーク・デーモン|Demon Chadarnook|Gefallener Chadarnook|Démon Chadarnouk) starts using (?:デモニックシアー|Demonic Shear|Dämonische Schere|Cisailles Démoniaques) on (\y{Name})/,
+
       alertText: function(data, matches) {
         if (matches[1] == data.me)
-          return 'Tank Buster on YOU';
+          return {
+			      en: 'Tank Buster on YOU',
+			      de: 'Tank Buster auf DIR',
+		      };
```
Comment (quisquous) [inline ui/raidboss/data/triggers/o6s.js]: Please indent the closing curly brace to line up with the return as it is elsewhere.

File: ui/eureka/eureka.html (hunk 1 above)
Comment (DrLippe) [reply ui/eureka/eureka.html]: this was just a test.
i zoomed the overlay to 60% and it supports the resize handle.
but a more overlay friendly display of the information would be nice.

File: ui/raidboss/data/triggers/o7s.js (hunk 2 above)
Comment (DrLippe) [reply ui/raidboss/data/triggers/o7s.js]: seems like even if you put notepad++ to space instead of tabs, the spaces are still tabs in disguise..

**Comment by quisquous** [comment]
I went ahead and landed all the 5, 6, and 7 translations that you did since I hadn't heard from you in a while.  Thanks for all the localization help!

**Comment by DrLippe** [comment]
sry about that, got a lot of work to do lately and since i'm playing mainly on ps4 i'm not that into this as i would be if i could actually use it ;)

**Pull request closed without merging**